submissions/Ant-C-tech/friends-app/js/app.js

```diff
@@ -0,0 +1,314 @@
+'use strict'
+
+const MAIN = document.querySelector('#main')
+const MUSIC_BUTTON = document.querySelector('#musicButton')
+const SEARCH = document.querySelector('.search')
+const SEARCH_INPUT = document.querySelector('#icon_prefix')
+const RESET_BUTTON = document.querySelector('.resetButton')
+
+const API_LINK = 'https://randomuser.me/api/'
+
+const FRIENDS_MIN = 30
+const FRIENDS_MAX = 50
+
+const APP_AUDIO = new Audio('./audio/you_ve_got_a_friend_in_me.mp3')
+APP_AUDIO.loop = true
+APP_AUDIO.volume = 0.3
+
+//Classes by AnimateCSS (default speed is 1s)
+const SHOW_ELEM_PRIMARY_ANIMATION = 'animate__zoomIn'
+const HIDE_ELEM_PRIMARY_ANIMATION = 'animate__zoomOut'
+const SHOW_ELEM_SECONDARY_ANIMATION = 'animate__fadeIn'
+const HIDE_ELEM_SECONDARY_ANIMATION = 'animate__fadeOut'
+const ANIMATION_SPEED = 'animate__faster' //500ms
+
+let FRIENDS_SOURCE
+let CURRENT_FRIENDS = []
+let IS_MUSIC_STOPPED_BY_USER = false
+let CURRENT_CONTENT
+let CURRENT_ANIMATION_HIDE
+let CURRENT_ANIMATION_SHOW
+let CURRENT_ANIMATION_SPEED
+
+
+initApp()
+
```

Comment:
Empty lines. Please, use prettier

Reply:
![settings-json](https://user-images.githubusercontent.com/59659904/103284461-f3b41000-49e3-11eb-87dc-56bef4e0bb27.jpg)
It seems to me I use something but it maybe needs some additional settings. In this specific case I made empty strings to stress initApp. I have removed them.

Reply:
`beautify` !== `prettier` :)

https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode